components/collective-navbar/index.js

```diff
@@ -290,6 +291,13 @@ const CollectiveNavbar = ({
   callsToAction = { ...getDefaultCallsToActions(collective, isAdmin), ...callsToAction };
   const isEvent = collective?.type === CollectiveType.EVENT;
 
+  const navbarRef = useRef();
+  useGlobalBlur(navbarRef, outside => {
+    if (!outside) {
+      setTimeout(() => setExpanded(false), 200);
+    }
+  });
+
```

Comment:
Actually the fix in https://github.com/opencollective/opencollective-frontend/pull/5555/files#diff-fdf6541d4ad9fe92e798118579cc5b59eaf80ddcbb4bc5d59638388b6d035aa9 probably makes this unnecessary. I'll merge it now, let me know if that fixes it for you